[2023-08-31 13:23:37,125] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2022-01-17T00:00:00+00:00 [queued]>
[2023-08-31 13:23:37,177] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2022-01-17T00:00:00+00:00 [queued]>
[2023-08-31 13:23:37,178] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:23:37,179] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:23:37,179] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:23:37,212] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-01-17T00:00:00+00:00
[2023-08-31 13:23:37,224] {standard_task_runner.py:52} INFO - Started process 5642 to run task
[2023-08-31 13:23:37,246] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2022-01-17T00:00:00+00:00', '--job-id', '520', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpwvy2tj_j', '--error-file', '/tmp/tmpne08bya5']
[2023-08-31 13:23:37,250] {standard_task_runner.py:77} INFO - Job 520: Subtask second_task
[2023-08-31 13:23:37,391] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2022-01-17T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:23:37,519] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-17T00:00:00+00:00
[2023-08-31 13:23:37,521] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:23:37,523] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:23:37,562] {bash.py:169} INFO - Output:
[2023-08-31 13:23:37,600] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:23:37,610] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:23:37,677] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220117T000000, start_date=20230831T132337, end_date=20230831T132337
[2023-08-31 13:23:37,922] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:23:37,953] {local_task_job.py:146} INFO - Task exited with return code 0
